packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -375,13 +396,14 @@ const VirtualListBaseFactory = (type) => {
 
 		jumpToSpottableItem = (keyCode, target) => {
 			const
-				{cbScrollTo, data, dataSize} = this.uiRef.props,
+				{isDisabledItem} = this.props,
+				{cbScrollTo, dataSize} = this.uiRef.props,
 				{firstIndex, numOfItems} = this.uiRef.state,
 				{isPrimaryDirectionVertical} = this.uiRef,
 				rtl = this.props.rtl,
 				currentIndex = Number.parseInt(target.getAttribute(dataIndexAttribute));
 
-			if (!data || !Array.isArray(data) || !data[currentIndex] || data[currentIndex].disabled) {
+			if (!isDisabledItem || isDisabledItem(currentIndex)) {
```

Comment:
Would it be better to assume that it's not disabled if the method is not supplied here?   `isDisabledItem && ...`

Reply:
@webOS101 
Do you mean that it is better to wrap the code in `jumpToSpottableItem()` with the following condition?

```
if (
	isItemDisabled &&
	!isItemDisabled(currentIndex)
) {
	// Moved the code here
}
```